mods/ra/metrics.yaml

```diff
@@ -16,3 +16,6 @@ Metrics:
 	IncompatibleProtectedGameColor: B22222
 	IncompatibleVersionColor: D3D3D3
 	TextfieldColorHighlight: 562020
+	ChatLine: ChatLine
```

Comment:
Can we rename this to `ChatLineSound`? (`ChatLineSound: ChatLine`)
Imho it is not clear what this does from just looking at it.